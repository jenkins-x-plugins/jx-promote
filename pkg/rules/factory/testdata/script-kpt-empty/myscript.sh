#!/bin/sh

# fetch resources

echo "DONE"