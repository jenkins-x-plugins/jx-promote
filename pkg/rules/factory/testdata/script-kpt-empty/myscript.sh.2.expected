#!/bin/sh

# fetch resources
kpt pkg get https://github.com/myorg/myapp.git/kubernetes@v1.2.4 $(FETCH_DIR)/namespaces/jx

echo "DONE"